### PR TITLE
Support platform registration, migrate to cloud.redhat.com, diverge legacy and platform

### PR DESCRIPTION
**Pull request by gravitypriest**
opening PR just to see what tests are failing/passing

### CODE REVIEW

File: insights/client/phase/v1.py [hunk 1]
```diff
@@ -127,13 +127,66 @@ def post_update(client, config):
     # create a machine id first thing. we'll need it for all uploads
     logger.debug('Machine ID: %s', client.get_machine_id())
     logger.debug("CONFIG: %s", config)
+
+    # -------delete everything below this line-------
```
Comment (Jason-RH) [inline insights/client/phase/v1.py]: Is everything below this line being deleted later?

Comment (gravitypriest) [reply insights/client/phase/v1.py]: yep. once the legacy path is phased out.

**Comment by gravitypriest** [comment]
test this please

**Comment by gravitypriest** [comment]
I think... it's ready now.

**Comment by gravitypriest** [comment]
test this please

**Pull request merged**